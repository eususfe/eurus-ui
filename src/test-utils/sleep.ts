/*
 * @Author: jiangxd
 * @Date: 2022-09-23 14:06:55
 * @LastEditTime: 2022-09-23 14:07:04
 * @LastEditors: jiangxd
 * @Description:
 * @FilePath: /eurus-ui/src/test-utils/sleep.ts
 */
const sleep = (time = 0) => new Promise(resolve => setTimeout(resolve, time));

export default sleep;
