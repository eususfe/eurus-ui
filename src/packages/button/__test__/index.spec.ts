import { shallowMount } from '@vue/test-utils'
import Button from '..'

describe('Button.vue', () => {
  test('render', () => {
    const wrapper = shallowMount(Button, {
      props: {},
    })
    expect(wrapper.classes()).toContain('e-button')
  })
})
