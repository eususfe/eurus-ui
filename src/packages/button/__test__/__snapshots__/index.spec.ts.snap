// Vitest Snapshot v1

exports[`Button.vue > button snapshot 1`] = `
"<button class=\\"e-button e-button--default bg-default text-black e-button--md\\" disabled=\\"false\\">
  <!---->
  <!---->
</button>"
`;

exports[`Button.vue > render 1`] = `
"<button class=\\"e-button e-button--default bg-default text-black e-button--md\\" disabled=\\"false\\">
  <!---->
  <!---->
</button>"
`;

exports[`button > button snapshot 1`] = `
"<button class=\\"e-button e-button--default bg-default text-black e-button--md\\" disabled=\\"false\\">
  <!---->
  <!---->
</button>"
`;
